src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -118,6 +118,29 @@ namespace winrt::TerminalApp::implementation
         _systemRowsToScroll = _ReadSystemRowsToScroll();
     }
 
+    bool TerminalPage::IsElevated() const noexcept
+    {
+        // use C++11 magic statics to make sure we only do this once.
+        // This won't change over the lifetime of the application
+
+        static const bool isElevated = []() {
+            // *** THIS IS A SINGLETON ***
+            auto result = false;
```

Comment:
Correct me if I'm wrong, but this is only executed once during start right? In that case I wouldn't make this a singleton.

Reply:
this is actually about to get removed by a4acdeb5f in #11222 when I next bump that PR